include/miopen/miopen.h

```diff
@@ -5663,24 +5663,52 @@ MIOPEN_EXPORT miopenStatus_t miopenSumForward(miopenHandle_t handle,
 
 #ifdef MIOPEN_BETA_API
 
-/*! @ingroup argmax
- * @brief Find the index of the maximum value of a tensor across dimensions.
+/*! @ingroup ReduceExtreme
+ * @enum miopenReduceExtremeOp_t
+ * Reduction Extreme operation types
+ */
+typedef enum
+{
+    MIOPEN_REDUCE_EXTREME_ARGMIN =
+        1, /*!< the operation is getting the minimum index of the reduced elements */
+    MIOPEN_REDUCE_EXTREME_ARGMAX =
+        2, /*!< the operation is getting the maximum index of the reduced elements */
+    MIOPEN_REDUCE_EXTREME_MIN =
+        3, /*!< the operation is getting the minimum value and index of the reduced elements */
+    MIOPEN_REDUCE_EXTREME_MAX =
+        4, /*!< the operation is getting the maximum value and index of the reduced elements */
+} miopenReduceExtremeOp_t;
+
+// ReduceExtreme APIs
+/** @addtogroup ReduceExtreme
+ *
+ *  @{
+ */
+
+/*! @brief Find the the extreme(minimum, maximum) value and index of a tensor across dimensions.
```

Comment:
Typo
```suggestion
/*! @brief Find the the extreme (minimum, maximum) value and index of a tensor across dimensions.
```
Now I see why it is called reduceExtreme 😄

Reply:
fixed it.

Reply:
[resolved]